tmp_translate_lm.py

```
@@ -0,0 +1,97 @@
+
+    # Give LM Studio time to spin up the model
+    print("Waiting 5s for model init...")
+    time.sleep(5)
+
+    for i, jp in enumerate(keys):
+        last_error = None
+        for attempt in range(6):
+            try:
+                out[jp] = translate_one(jp, session)
+                break
+            except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)